docs/api/class-reference.md

```diff
@@ -181,6 +181,11 @@ This object represents a generic Creative. It's used as a parent object for more
 - `attributes: Object`
 - `children: Array<Extension>`
 
+## UniversalAdId<a name="universaladid"></a>
+
+- `idRegistry: String|unknown`
+- `value: any`
```

Comment:
For `idRegistry` as the default value is a string, I think that we should put `String|"unknown"`  or add the description from IAB specs : `A string used to identify the URL for the registry website where the unique
creative ID is cataloged. Default value is “unknown”.` .
Also `value` is a String